metabase/metabase
# Fix combined cards crash when cards have invalid settings

Closes https://github.com/metabase/metabase/issues/45114

### Description

When a dashcard combines multiple cards and the cards have invalid `graph.metrics` settings we rendered an error instead of falling back to computing the default `graph.metrics`.

### How to verify & Demo

This loom shows that it works and describes how to reproduce it locally
https://www.loom.com/share/be81f926707c4c288a98ffc0d6e93879?sid=244e47b7-7fcf-4fa7-813f-c6feddb87c8a

- Create 2 compatible questions that use line/area/bar/combo charts
- Combine them on a dashcard
- Go to app db and put some invalid value into `graph.metrics` for both of them
- The dashcard should still work

### Checklist
